### PR TITLE
Add basic Module serialization BC test

### PR DESCRIPTION
**Pull request by albanD**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #96311
* #96310
* __->__ #96238

### CODE REVIEW

**Review by ezyang — APPROVED** [review]

File: test/load_torchscript_model.py [hunk 1]
```diff
@@ -2,5 +2,9 @@
 import torch
 
 if __name__ == '__main__':
-    print(torch.jit.load(sys.argv[1]))
+    script_mod = torch.jit.load(sys.argv[1])
+    mod = torch.load(sys.argv[1] + ".orig")
+    print(script_mod)
```
[the remark this answers is not in the document]
Comment (albanD) [reply test/load_torchscript_model.py:7]: It was there before. So I left it.

[the remark this answers is not in the document]
Comment (ezyang) [reply test/load_torchscript_model.py:7]: s'fine

**Comment by wconstab** [comment]
cool, thanks for adding @albanD!

**Pull request closed without merging**